samples/sdl_ttf/main.c

```diff
@@ -0,0 +1,241 @@
+/*
+  Copyright (C) 1997-2016 Sam Lantinga <slouken@libsdl.org>
```

Comment:
Is this valid? There's at least some changes for nxdk (would possiby need someone elses copyright head) and I'm not sure how much is actually taken from SDL (maybe we can remove the entire copyright head).

I don't know how the copyright head looks for other samples, but I feel like there shouldn't be one, or it should be CC0 or something to signal intend (so people don't copy copyright headers from some sample into every program).

Reply:
It probably needs some modification, sample is just a slightly modified version of the already available sdl one.
I'll be the first to admit that I don't feel comfortable when it comes to licenses, I'm not sure what modification should be done for this to be valid.